src/views/Prediction/Prediction.js

```diff
@@ -0,0 +1,147 @@
+import React, {Component} from 'react';
+import {connect} from 'react-redux';
+import PropTypes from 'prop-types';
+import {CLAS_MODEL_CHANGED, MODEL_CHANGED, modelsRequested, REG_MODEL_CHANGED} from '../../actions/ModelActions';
+import {LOG_CHANGED, logListRequested} from '../../actions/LogActions';
+import {jobsRequested} from '../../actions/JobActions';
+import {JOB_RUN_CHANGED, submitPrediction} from '../../actions/RuntimeActions';
+import LogSelector from '../../components/prediction/LogSelector';
+import ResultTable from '../../components/prediction/ResultTable';
+import {fetchStatePropType, jobRunPropType, logsStore, modelPropType} from '../../propTypes';
+import {modelsToString} from '../../util/dataReducers';
+import {splitsRequested} from '../../actions/SplitActions';
+import {mapJobs} from '../../util/unNormalize';
+import {CardText} from 'react-md/lib/Cards/index';
+import ModelSelector from '../../components/prediction/ModelSelector';
+import {REGRESSION} from '../../reference';
+import {Card} from 'react-md';
+
+class Prediction extends Component {
+  onChangeLog(logId) {
+    const log = this.props.logs.byId[logId];
+    const pLength = log.properties.maxEventsInLog;
+    this.props.onLogChange(logId, pLength);
+    this.props.onChangeJRun(logId);
+  }
+
+  onModelChange({method}, modelId) {
+    if (method === REGRESSION) {
+      this.props.onRegModelChange(modelId);
```

Comment:
here i changed stuff a bit. but do you really need 2 method like `onRegModelChange`  and `onModelChange`? One should be enough